README.md

```diff
@@ -28,15 +28,15 @@ One button below has an `ng-click` attribute; the other has `data-ng-click` inst
 ```
 
 ```text
-Your answer goes here...
+"data" is used for HTML validation on the app. If you run "ng-click" through an html validator service, it will not pass as compliant.
 ```
 
 ### Question 3
 
 Which of the three following options demonstrates the best usage of `ng-app`? **Explain your answer.**
 
 ```text
-Your answer goes here...
+"ng-app" directive is used as a starting point of an angular application, initializing the AngularJS framework. Option A is correct, as the ng-app directive should be placed at the root of an HTML document.
 ```
```

Comment:
You are right, but I want to mention that the `ng-app` directive is used for where you want that particulars modules functionality.